lib/Screens/About/about_us.dart

```diff
@@ -40,46 +19,165 @@ class _AboutState extends State<About> {
         child: Center(
           child: Column(
             children: <Widget>[
-              Container(
-                decoration: BoxDecoration(
-                  border: Border.all(color: Colors.orange, width: 3),
-                  borderRadius: BorderRadius.circular(5.0),
-                ),
-                child: Column(
-                  children: [
-                    createTextBox(
-                        "A loving and joyful multicultural church community in the heart of Bankstown, Sydney."),
-                    createTextBox(
-                        "Kingdom people who are led by God's Word and the Holy Spirit."),
-                    createTextBox(
-                        "Children of God and believers in His goodness to us all the time."),
-                    createTextBox(
-                        "Passionate about bringing the message of reconciliation, freedom, healing and deliverance to all people."),
-                    createTextBox(
-                        "We would love to see you online or in person at our 10:00am service on Sunday mornings."),
-                  ],
-                ),
+              const Padding(
+                padding:
+                    EdgeInsets.only(left: 25.0, right: 25, bottom: 25, top: 50),
+                child: Center(
+                    child: Text('About US',
```

Comment:
Please remove this widget and instead, make it an app bar.

Reply:
Okay